v2/test/pre-release/pre_release_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package multitenant_test
+
+import (
+	"testing"
+
+	network "github.com/Azure/azure-service-operator/v2/api/network/v1beta20201101"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+	"github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	resourceGroupName     = "aso-sample-rg"
```

Comment:
minor: I think you should use a name like `aso-upgrade-test-rg` or something. Not a name that we already use in samples. Less risk of us colliding with ourselves.

Reply:
Actually, you should also use the `TEST_LIVE_RESOURCE_PREFIX` on this resource, which is `asolivetest`, that way if we do for some reason end up leaking this resource, the ` cleanup-old-live-azure-resources` step in the Taskfile which is already run periodically will clean it up.